### PR TITLE
Fixed invalid links throughout the readme files

### PR DESCRIPTION
**Pull request by Hartmannlight**
## Description

I tested all the links in the readme files and if they didn't work, I replaced them with the valid links that were meant to be used

## Motivation and Context

Contribution to maintain the quality of the samples

## How Has This Been Tested?

I went through the READMEs and clicked on the new links to see if they actually worked

## Types of changes

- [x] Improvement (non-breaking change that improves a sample)
- [ ] Bug fix (fixes an issue with a current sample)
- [ ] New Sample (adds a new sample)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Hartmannlight** [comment]
@dotnet-policy-service agree

**Comment by Hartmannlight** [comment]
The repository also contains the following links, which are not accessible:

http://docs.nanoframework.net/api/nanoFramework.GiantGecko.Adc.html
http://docs.nanoframework.net/api/nanoFramework.Hardware.GiantGecko.Utilities.html
http://docs.nanoframework.net/api/nanoFramework.System.Collection.Hashtable.html
http://docs.nanoframework.net/api/Windows.Devices.Gpio.html
http://docs.nanoframework.net/api/System.Device.UsbClient.html

I couldn't find the documentation myself, and I'm not sure why it's missing, so I couldn't replace these links.
I hope they are still valid and that you just need to update the documentation to make everything work.

**Comment by networkfusion** [comment]
@Hartmannlight Here is a sample fix for the Usb Client: https://github.com/nanoframework/nanoframework.github.io/pull/186

Note that the Windows.Decices namespace is deprecated, so it might be worth finding those links and noting as such.

The [nanoFramework.System.Collection.Hashtable.html](http://docs.nanoframework.net/api/nanoFramework.System.Collection.Hashtable.html) should actually be `collections`, so if you can track that down and fix, it would be helpful.

File: samples/1-Wire/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you have another type of device supporting 1-Wire, you will have to adjust th
 
 ### Reference
 
-- [nanoFramework.Devices.OneWire](http://docs.nanoframework.net/api/nanoFramework.Devices.OneWire.html)
+- [nanoFramework.Devices.OneWire](http://docs.nanoframework.net/api/nanoFramework.Device.OneWire.html)
```
Comment (networkfusion) [inline samples/1-Wire/README.md:31]:
```suggestion
- [nanoFramework.Device.OneWire](http://docs.nanoframework.net/api/nanoFramework.Device.OneWire.html)
```

**Comment by Hartmannlight** [comment]
- [ ] nanoFramework.GiantGecko.Adc
I think, you can add [this](https://github.com/nanoframework/nanoFramework.GiantGecko.Adc) to [your pull request](https://github.com/nanoframework/nanoframework.github.io/pull/186) @networkfusion 

- [ ] nanoFramework.Hardware.GiantGecko.Utilities
It looks like it does not exist

- [x] nanoFramework.System.Collection.Hashtable
- [x] Windows.Devices.Gpio
- [x] System.Device.UsbClient

**Comment by josesimoes** [comment]
@Hartmannlight URLs for all missing Gecko libraries have been added. Feel free to use and update as needed. 😉

**Review by josesimoes — APPROVED** [review]
Thanks for fixing all these! 👍🏻

**Pull request merged**